recipes/ffmpeg/all/conanfile.py

```diff
@@ -376,7 +385,11 @@ def _configure_autotools(self):
                 # Visual Studio 2013 (and earlier) doesn't support "inline" keyword for C (only for C++)
                 self._autotools.defines.append("inline=__inline")
         if tools.cross_building(self):
-            args.append("--target-os={}".format(self._target_os))
+            if self._target_os == "emscripten":
+                args.append("--target-os=none")
+            else:
+                args.append("--target-os={}".format(self._target_os))
```

Comment:
Would changing the code to let `self._target_os` return `none` when `os=Emscripten` make more sense?

Reply:
that might be too complicated - this part of the code is too sophisticated and fragile. I think it's fine.